app/views/AnalyticalFramework/FrameworkForm/PrimaryTagging/SectionsEditor/styles.css

```diff
@@ -1,18 +1,55 @@
 .form {
     display: flex;
     flex-direction: column;
+    background: var(--dui-color-foreground);
     overflow-y: auto;
 
-    .content {
-        overflow-y: auto;
+    .main-container {
+        background: var(--dui-color-foreground);
+        padding: 0;
+
+        .content {
+            overflow-y: auto;
+
+            .section-container {
+                background: var(--dui-color-foreground);
+                padding: 0;
+
+                .section-expandable {
+                    padding: var(--dui-spacing-medium);
+
+                    .section-header {
+                        display: flex;
+                        background-color: var(--dui-color-background-information);
+                        grid-gap: var(--spacing);
+
+                        .heading {
+                            padding: calc(var(--spacing) / 2) 0;
+                        }
+                    }
+                    .text-input {
+                        margin: var(--dui-spacing-small) 0;
+                    }
+                }
+            }
+        }
     }
```

Comment:
Should we need to manually modify padding for these many components? @frozenhelium

Reply:
Yes, it's needed to achieve the look we're going for!